pytest.ini

```diff
@@ -26,6 +26,6 @@ addopts = --live-server-port=5000
 live_server_scope = function
 
 # run tests only from `tests` directory
-# we do not want to run tests of the converter submodule (in yaptite/converter)
+# we do not want to run tests of the converter git dependency (in .venv/src/converter)
 # as the converter tests assume to be run in a standalone converter repo
```

Comment:
```suggestion
# we do not want to run tests in any other directories
# we are explicitit to avoid any confusion on test layout
# as in https://docs.pytest.org/en/stable/explanation/goodpractices.html#conventions-for-python-test-discovery
```